toplevel/toploop.ml

```diff
@@ -414,6 +414,7 @@ let loop ppf =
       Buffer.reset phrase_buffer;
       Location.reset();
       first_line := true;
+      Lexer.reset_syntax_mode();
```

Comment:
This looks to me like it would require you to re-turn on quotations for every phrase you wrote in the REPL. That would not be ideal.